### PR TITLE
Docs only removal of NGSI-v1

### PR DESCRIPTION
**Pull request by jason-fox**
Attempt to reduce the scope of #995 - just updating docs

### CODE REVIEW

File: doc/howto.md [hunk 1]
```diff
@@ -494,26 +494,13 @@ Postman-Token: 993ac66b-72da-9e96-ab46-779677a5896a
...
-    "entities": [
-        {
-            "isPattern": "true",
-            "id": ".*",
-            "type": "BasicULSensor"
-        }
-    ],
-    "attributes" : [
-    	"l"]
-}
```
Comment (fgalan) [inline doc/howto.md:516]: Instead of deleting this, it should be change to a NGSI-v2 equivalent payload.

**Comment by mapedraza** [comment]
Makes sense to remove all references to NGSI-v1 in the documentation like here:
https://github.com/telefonicaid/iotagent-node-lib/blob/3f4491019b0284e166a77e950bc7d927d44ff0c4/doc/advanced-topics.md#geojson-support-this-only-applies-for-ngsi-ld-not-for-ngsi-v1-and-ngsi-v2

Take into account that also appears on the ToC at the top of the document

File: doc/howto.md (hunk 1 above)
Comment (jason-fox) [reply doc/howto.md]: Fixed 6d83334

**Comment by jason-fox** [comment]
> Makes sense to remove all references to NGSI-v1 in the documentation like here:

Fixed Fixed 6d83334

**Review by fgalan — APPROVED** [review]
LGTM

Passing the ball to @mapedraza for final LGTM and merge, as he has only some comments on the PR.

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**